sdk/communication/AzureCommunication/Source/CommunicationIdentifierModel.swift

```diff
@@ -0,0 +1,82 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+import AzureCore
+import Foundation
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable identifier_name
+// swiftlint:disable line_length
+// swiftlint:disable cyclomatic_complexity
+
+/// Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user.
+public struct CommunicationIdentifierModel: Codable {
+    // MARK: Properties
+
+    /// Kind of the communication identifier.
+    public let kind: CommunicationIdentifierKind
+    /// Full Id of the identifier.
+    public let id: String?
+    /// The phone number in E.164 format.
+    public let phoneNumber: String?
+    /// The AAD object Id of the Microsoft Teams user.
+    public let microsoftTeamsUserId: String?
+    /// True if the identifier is anonymous.
+    public let isAnonymous: Bool?
+
+    // MARK: Initializers
+
+    /// Initialize a `CommunicationIdentifierModel` structure.
+    /// - Parameters:
+    ///   - kind: Kind of the communication identifier.
+    ///   - id: Full Id of the identifier.
+    ///   - phoneNumber: The phone number in E.164 format.
+    ///   - microsoftTeamsUserId: The AAD object Id of the Microsoft Teams user.
+    ///   - isAnonymous: True if the identifier is anonymous.
+    public init(
+        kind: CommunicationIdentifierKind, id: String? = nil, phoneNumber: String? = nil,
+        microsoftTeamsUserId: String? = nil, isAnonymous: Bool? = nil
+    ) {
+        self.kind = kind
+        self.id = id
+        self.phoneNumber = phoneNumber
+        self.microsoftTeamsUserId = microsoftTeamsUserId
+        self.isAnonymous = isAnonymous
+    }
+
+    // MARK: Codable
+
+    enum CodingKeys: String, CodingKey {
```

Comment:
Do you need to set this? 
It looks like you there is no change in coding keys names.

Reply:
If we change `id` to `identifier` we will need to set coding keys.

Reply:
Enumerations.swift and CommunicationIdentifierModel.swift are generated code from AutoRest code.